pkg/apis/machine/types.go

```diff
@@ -515,13 +536,23 @@ type RollingUpdateMachineDeployment struct {
 	// Absolute number is calculated from percentage by rounding up.
 	// By default, a value of 1 is used.
 	// Example: when this is set to 30%, the new MC can be scaled up immediately when
-	// the rolling update starts, such that the total number of old and new machines do not exceed
+	// the update starts, such that the total number of old and new machines do not exceed
 	// 130% of desired machines. Once old machines have been killed,
 	// new MC can be scaled up further, ensuring that total number of machines running
 	// at any time during the update is atmost 130% of desired machines.
 	MaxSurge *intstr.IntOrString
 }
 
+// OrchestrationType specifies the orchestration type for the inplace update.
+type OrchestrationType string
+
+const (
+	// OrchestrationTypeAuto signifies that the machines are selected automatically based on MaxUnavailable for being updated.
```

Comment:
```suggestion
	// OrchestrationTypeAuto signifies that the machines are automatically selected for update  based on UpdateConfiguration .
```

Reply:
Addressed in [2fdffb1](https://github.com/gardener/machine-controller-manager/pull/962/commits/2fdffb1978dcb47f3a2801d5e330c1189a2baf8a). PTAL.